autofit/mapper/prior/prior.py

```diff
@@ -346,6 +358,22 @@ def value_for(self, unit):
         """
         return self.lower_limit + unit * (self.upper_limit - self.lower_limit)
 
+    def log_prior_from_value(self, value):
+        """Compute the log prior of a physical value, so the log likelihood of a model evaluation can be converted to a
+        posterior as log_prior + log_likelihood.
+
+        This is used by Emcee in the log likelihood function evaluation.
+
+        NOTE: For a UniformPrior this is always zero, provided the value is between the lower and upper limit. Given
+        this is check for when the instance is made (in the *instance_from_vector* function), we thus can simply return
```

Comment:
We did turn that check off for the aggregator. It was only to account for floating point errors but I wonder if there's a risk associated with not implementing this function properly?

If one reused it in a way that meant the assertion was not run then it would give incorrect results.

Reply:
This is a different thing - we are computed the log of the priors of the parameters. For a UniformPrior log_prior() = 0.0 for any value, so the function seems a bit odd. The comment is just explaining that.